### DISCUSSION TITLE
GitHub Runners Status: Requires Admin?

### DISCUSSION THREAD

**Discussion by alexlo03** [community/community#12508] [Enterprise]
Hello,
We have an enterprise pool of Actions Runners - our engineering team cannot see if they are running into problems with the Actions Runners because the status is not visible to them. To just see the STATUS of the runners requires Admin per
https://docs.github.com/en/actions/hosting-your-own-runners/monitoring-and-troubleshooting-self-hosted-runners#checking-the-status-of-a-self-hosted-runner

Can this be updated? Can we allow access to organization members? We have engineers who want to see if they are running into a problem with our actions runners pool and can't see anything.

Alternatively I would like the source data be exposed to DataDog/metrics so I can build dashboards for my users.

To compare to other tooling in the industry: Jenkins would show builder resources to Jenkins users for >10 years.

Thanks, Alex (from my non Enterprise account)

**Comment by ethomson** [comment]
Thanks @alexlo03 - I agree that this is an area that we need to improve. We have some work on the backlog to tackle this in the future, though I don't have an ETA. We'll keep this in mind when we start working on that.

Thanks much for taking the time to give us feedback.
